packages/subgraph/src/packages/token/governance-erc20.ts

```diff
@@ -12,8 +16,11 @@ function getOrCreateMember(user: Address, pluginId: string): TokenVotingMember {
   if (!member) {
     member = new TokenVotingMember(id);
     member.address = user;
-    member.plugin = pluginId;
     member.balance = BigInt.zero();
+    member.plugin = pluginId;
+
+    member.delegatee = id; // we assume by default member delegates itself
```

Comment:
Isn't it simpler to leave this empty? If there is no delegatee, it's also clear that you are the voter